modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -923,8 +927,9 @@ private void _collectObjectFields(
 					for (Method method : methodsTreeSet) {
 						GraphQLFieldDefinition field =
 							_liferayGraphQLFieldRetriever.getField(
-								deprecated, method,
-								processingElementsContainer);
+								deprecated,
+								_getDefaultGraphQLNamespace(versionedPath),
```

Comment:
You are always calculating the graphQLNamespace even if it is not going to be used. Maybe better just passing the path and calculating the namespace inside the method if it is needed